packages/oid4vci-rest-client/src/agent/OID4VCIRestClient.ts

```diff
@@ -0,0 +1,65 @@
+import { fetch } from 'cross-fetch'
+import {
+  IOID4VCIRestClient,
+  IVCIClientCreateOfferUriRequest,
+  IVCIClientCreateOfferUriRequestArgs,
+  IVCIClientCreateOfferUriResponse,
+} from '../types/IOID4VCIRestClient'
+import Debug from 'debug'
+import { IAgentPlugin } from '@veramo/core'
+
+const debug = Debug('sphereon:ssi-sdk.oid4vci-rest-client')
+
+/**
+ * @beta
+ */
+export class OID4VCIRestClient implements IAgentPlugin {
+  readonly methods: IOID4VCIRestClient = {
+    vciClientCreateOfferUri: this.vciClientCreateOfferUri.bind(this),
+  }
+
+  private readonly baseUrl?: string
```

Comment:
baseUrl vs baseUrl in interfaces. a url is a uri, a uri is not per definition a url

Reply:
This is the agent url, so always a URL. Better name would be agentBaseURL

Reply:
changed it to `agentBaseURL`